docs/tutorials/backport.md

```diff
@@ -1,21 +1,33 @@
 !!! warning "This tutorial might be partial or incomplete. Please check with the team if you have questions."
 
-Before the existance of the ETL, datasets were directly uploaded to our Grapher admin site. Sometimes, we want to use these datasets in ETL. However, they are not available by default as they were never imported via the ETL. To make them available in ETL we use _backport_.
+Before ETL came along, datasets were uploaded directly to our Grapher admin site. If you want to leverage these datasets in the ETL, they won't be available out of the box, since they weren't initially imported via ETL. You have two main options to bring them into ETL:
 
+1. Migrate them into the ETL pipeline.
+2. Move them to Fast-Track.
 
-The first step is getting them to Snapshot using:
+Choose Fast-Track if the dataset requires author edits. Otherwise, it's more straightforward to migrate to ETL.
+
+
+## How to Migrate to ETL
+
+Automatically generate ETL steps for a specific dataset (xxx) with the following command:
 
 ```
-bulk_backport
+ENV=.env.prod backport-migrate --dataset-id xxx --namespace your_namespace --version your_version --short-name your_short_name
```

Comment:
I'd suggest renaming `backport-migrate` -> `etl-backport-migrate`. It's easier at least for me to be in the ETL poetry shell and type `etl-` and tab, to remember the options available. But up to you.

Reply:
Can you please explain what `xxx` and `short_name` mean?

Reply:
Yeah, the naming isn't great. I'd rather do it as part of a different PR though (we could even use subcommands like `etl backport-migrate`).

> Can you please explain what xxx and short_name mean?

Clarified that. It's just dataset id and choosing short_name is up to you.